app/views/spree/shared/_quantity_select.html.erb

```diff
@@ -1,6 +1,11 @@
 <%# TODO: Use in cart %>
-<div class="d-inline-flex quantity-select">
-  <%= button_tag '-', type: 'button', class: "border-right-0 flex-grow-0 flex-shrink-0 py-0 px-3 quantity-select-decrease" %>
-  <%= number_field_tag input_name, 1, min: 1, max: maximum_quantity, class: "p-0 flex-grow-1 flex-shrink-1 text-center form-control border-left-0 border-right-0 quantity-select-value", 'aria-label': Spree.t('pdp.quantity') %>
-  <%= button_tag '+', type: 'button', class: "border-left-0  flex-grow-0 flex-shrink-0 py-0 px-3 quantity-select-increase" %>
+<!-- <div class="d-inline-flex quantity-select">
+  <%#= button_tag '-', type: 'button', class: "border-right-0 flex-grow-0 flex-shrink-0 py-0 px-3 quantity-select-decrease" %>
+  <%#= number_field_tag input_name, 1, min: 1, max: maximum_quantity, class: "p-0 flex-grow-1 flex-shrink-1 text-center form-control border-left-0 border-right-0 quantity-select-value", 'aria-label': Spree.t('pdp.quantity') %>
+  <%#= button_tag '+', type: 'button', class: "border-left-0  flex-grow-0 flex-shrink-0 py-0 px-3 quantity-select-increase" %>
+</div> -->
+<div class=" quantity-select">
+  <%= number_field_tag input_name, 1, min: 1, max: maximum_quantity, class: "p-0 flex-grow-1 flex-shrink-1 text-center form-control border-left-10 border-right-10 quantity-select-value ", 'aria-label': Spree.t('pdp.quantity') %>
```

Comment:
Its a dropdown in template file but we added a text field. 

Also we need to handle an edge case of 10 + option. Please discuss it with me.

Reply:
Will see later and make new ticket for this